src/routes/Routes.tsx

```diff
@@ -235,6 +236,16 @@ const Routes = () => {
             backRoute: `/colony/${colonyName}`,
           })}
         />
+        <AlwaysAccesibleRoute
+          exact
+          path={DECISIONS_PAGE_ROUTE}
+          component={ActionsPage}
+          layout={NavBar}
+          routeProps={({ colonyName }) => ({
+            backText: '',
```

Comment:
Text here should be "Go back" in order to be consistent with the Preview screen.

Reply:
This is consistent with Actions page and works without the extra text
<img width="607" alt="Screenshot 2022-09-20 at 18 31 24" src="https://user-images.githubusercontent.com/34057551/191313972-3d25d8f7-b863-41d1-a195-a48450428de9.png">
<img width="934" alt="Screenshot 2022-09-20 at 18 31 47" src="https://user-images.githubusercontent.com/34057551/191314041-4657fe21-4cf5-48dc-9c64-00ed9b970617.png">